_posts/2019-02-26-data-security-of-salesforce.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Data Security Of Salesforce"
+date:   2019-02-26 22:27:00 +0800
+categories: blog
+author: Harry Hou
+---
+
+[Articles](https://trailhead.salesforce.com/content/learn/modules/data_security)
+
+## Data Security of Salesforce
+
+As a CRM application, it's important for the user that giving the right people access to the right data, this post will introduce how to control those stuff things in Salesforce.
+
+### Levels of data access
+
+In Salesforce you can control which users have access to which data in your **whole org**, a specific **object**, a specific **field**, or an individual **record**.
+
+#### Control Access to the Organization
+
+At the organization level's data security there has one thing you can do is that ensure only employees who meet certain criteria can log in to Salesforce. In Salesforce you can do this by managing authorized users, setting password policies, and limiting when and where users can log in.
+
+#### Control Access to Objects
+
+In Salesforce an object is a collection of records, like leads or contacts. So how to control access to objects? Here we must take about **profile**.
```

Comment:
take about => talk about
profile => profiles (generally you use plural form here)